source/options.css

```diff
@@ -130,6 +131,17 @@ h2 ~ h2 {
 	margin-left: 0.6em;
 }
 
+.feature-new .feature-name::after {
+	content: 'New';
+	color: var(--feature-new);
+	font-size: 14px;
```

Comment:
Use `em` everywhere (except `border-width`), this text could be changed by the browser, we inherit their style.